### ISSUE TITLE
<Delay in PlayerView when share a url local>

### ISSUE THREAD

**Issue by sebasgarcia29** [bitmovin/bitmovin-player-react-native#130]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Sometimes in Android when loading a local video in the player it is not load, This issue no happen ever but may feel bad behavior in the UX.

I share the code whit an example and a video with the evidence.

```
import React, { useEffect, useCallback } from 'react';
import { StyleSheet, Platform } from 'react-native';
import {
  usePlayer,
  SourceType,
  PlayerView,
} from 'bitmovin-player-react-native';
import Config from 'react-native-config';

export const Video = () => {
  const player = usePlayer({
    licenseKey: Config.BITMOVIN_LICENSE_KEY,
  });

  useEffect(() => {
    player.load({
      url: 'http://127.0.0.1:52989/storage/emulated/0/Android/data/com.pentheraexample/files/virtuoso/media/0708724C-D140-43FE-9002-0B2B5C1C5DE6/playlist.mpd',
      type: Platform.OS === 'ios' ? SourceType.HLS : SourceType.DASH,
      title: 'Test title',
      poster: 'https://bitmovin-a.akamaihd.net/content/MI201109210084_1/poster.jpg',
      isPosterPersistent: false,
    });
    return () => {
      player.destroy();
    };
  }, [player]);

  const onReady = useCallback(
    (event) => {
      // Start playback
      player.play();
      // Print event timestamp
      console.log(event.timestamp);
    },
    [player]
  );

  const onSourceError = useCallback((event) => {
    console.log({ 'onSourceError: ': event });
  }, []);

  const onSourceLoaded = useCallback((event) => {
    console.log({ 'onSourceLoaded: ': event });
  }, []);

  return (
    <PlayerView
      style={styles.flex1}
      player={player}
      onReady={onReady}
      onSourceError={onSourceError}
      onSourceLoaded={onSourceLoaded}
    />
  );
};

const styles = StyleSheet.create({
  flex1: {
    flex: 1,
  },
});

```

### Expected Behavior

It is expected that whenever a local url is passed the video can be played normally.

### What platform(s) are you experiencing the issue on?

- [X] Android
- [ ] Android TV / Fire TV
- [ ] iOS
- [ ] tvOS

### Player React Native SDK version

0.6.0

### Stream URL (Optional)

http://127.0.0.1:52989/storage/emulated/0/Android/data/com.pentheraexample/files/virtuoso/media/0708724C-D140-43FE-9002-0B2B5C1C5DE6/playlist.mpd

### Additional information / Code snippets / Screenshots

I attach evidence with this behavior.

https://github.com/bitmovin/bitmovin-player-react-native/assets/44488463/5d3b4cbf-0e62-4921-8174-2f46e7921435

**Comment by jonathanm-tkf** [comment]
Hi guys any news on this ticket?

thanks and regards.

**Comment by christophaigner** [comment]
Hi @sebasgarcia29 & @jonathanm-tkf,

Thanks for reporting this issue & apologies for the delay here. This looks very similar to #163.
We've recently released [v0.8.0](https://github.com/bitmovin/bitmovin-player-react-native/releases/tag/v0.8.0), which contains a fix/workaround for this (see https://github.com/bitmovin/bitmovin-player-react-native/pull/174 for more details).

Can you still reproduce this with the latest version?

Chris

**Comment by jonathanm-tkf** [comment]
Hi @christophaigner  currently still happening in the latest version, unfortunately.
